Jenkinsfile

```diff
@@ -520,6 +520,7 @@ def e2e(Map args = [:]) {
     def goVersionForE2E = readFile('.go-version').trim()
     withEnv(["GO_VERSION=${goVersionForE2E}",
               "BEATS_LOCAL_PATH=${env.WORKSPACE}/${env.BASE_DIR}",
+              "BEAT_VERSION=${env.VERSION}-SNAPSHOT",
```

Comment:
Only for clarity, I add this comment, it's nothing about the code review :

env.VERSION is set in  -> https://github.com/elastic/beats/blob/295bbb686e7320c4b68b3e8399e079b17d410e66/Jenkinsfile#L75-L79

Reply:
We talked online and the GITHUB_CHECK_SHA1 is the one responsible of passing the context about current commit, so the aforementioned file does not apply here, as we are achieving the same.